src/cms/package.json

```diff
@@ -13,36 +13,42 @@
     "loaddates": "node scripts/refresh-dates.js"
   },
   "dependencies": {
-    "@_sh/strapi-plugin-ckeditor": "^2.0.4",
-    "@babel/plugin-transform-block-scoping": "^7.23.0",
-    "@ckeditor/ckeditor5-markdown-gfm": "^36.0.1",
+    "@_sh/ckeditor5-font-with-picker": "0.0.1",
+    "@_sh/strapi-plugin-ckeditor": "^2.1.1",
+    "@babel/plugin-transform-block-scoping": "^7.24.7",
+    "@ckeditor/ckeditor5-markdown-gfm": "^41.4.2",
     "@fortawesome/fontawesome-free": "^5.15.4",
-    "@opensearch-project/opensearch": "^2.4.0",
-    "@strapi/plugin-documentation": "4.14.4",
-    "@strapi/plugin-graphql": "4.14.4",
-    "@strapi/plugin-users-permissions": "4.14.4",
-    "@strapi/provider-email-nodemailer": "4.14.4",
-    "@strapi/provider-upload-aws-s3": "4.14.4",
-    "@strapi/strapi": "4.14.4",
+    "@opensearch-project/opensearch": "^2.10.0",
+    "@strapi/plugin-documentation": "4.25.1",
+    "@strapi/plugin-graphql": "4.25.1",
+    "@strapi/plugin-users-permissions": "4.25.1",
+    "@strapi/provider-email-nodemailer": "4.25.1",
+    "@strapi/provider-upload-aws-s3": "4.25.1",
+    "@strapi/strapi": "4.25.1",
     "codemirror": "5.65.12",
     "jwks-rsa": "^2.1.5",
     "pg": "^8.11.3",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-router-dom": "^5.3.4",
     "semver": "^7.5.4",
-    "strapi-plugin-redis": "^1.0.1",
+    "strapi-plugin-redis": "^1.1.0",
     "strapi-plugin-rest-cache": "^4.2.8",
     "strapi-plugin-transformer": "^3.1.2",
-    "strapi-provider-rest-cache-redis": "^4.2.8"
+    "strapi-provider-rest-cache-redis": "^4.2.9",
+    "styled-components": "^5.3.11",
+    "yup": "^0.32.11"
   },
   "engines": {
     "node": ">=14.19.1 <=18.x.x",
     "npm": ">=6.0.0"
   },
   "devDependencies": {
-    "eslint": "^8.51.0",
+    "better-sqlite3": "^11.0.0",
+    "eslint": "^8.57.0",
     "eslint-config-prettier": "^8.10.0",
     "jest": "^29.7.0",
-    "sqlite3": "^5.1.4",
-    "supertest": "^6.3.3"
+    "supertest": "^6.3.4"
   },
```

Comment:
Just curiosity, is there any reason to change `sqlite3` to `better-sqlite3`?

Reply:
The tests on Github failed when I first created my PR.  
The error was `Cannot find module 'better-sqlite3'`
https://github.com/bcgov/bcparks.ca/actions/runs/9670320509/job/26678821113